node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/ClusterNodesTimeseries.java

```diff
@@ -45,15 +51,25 @@ private ClusterNodesTimeseries(NodeList clusterNodes, List<NodeTimeseries> times
         this.timeseries = timeseries;
     }
 
+    public String description() {
+        return "initial measurements: " + initialCount +
+               ", after warmpup: " + afterWarmupCount +
+               ", after stable: " + afterStableCount +
+               ", final measurements: " + finalCount;
+    }
+
     public boolean isEmpty() {
         return measurementsPerNode() == 0;
     }
 
     /** Returns the average number of measurements per node */
     public double measurementsPerNode() {
         if (clusterNodes.size() == 0) return 0;
-        int measurementCount = timeseries.stream().mapToInt(m -> m.size()).sum();
-        return (double)measurementCount / clusterNodes.size();
+        return (double) totalMeasurementsIn(timeseries) / clusterNodes.size();
```

Comment:
```suggestion
        return (double) finalCount / clusterNodes.size();
```

Reply:
I'm planning to remove the counters later.